src/api/helpers/get-populated-export-contract/get-populated-agent/index.test.ts

```diff
@@ -0,0 +1,115 @@
+import getPopulatedAgent from '.';
+import getKeystoneContext from '../../../test-helpers/get-keystone-context';
+import agent from '../../../test-helpers/export-contract-agent';
+import service from '../../../test-helpers/export-contract-agent-service';
+import serviceCharge from '../../../test-helpers/export-contract-agent-service-charge';
+import { ApplicationExportContractAgent, ApplicationExportContractAgentService, ApplicationExportContractAgentServiceCharge, Context } from '../../../types';
+
+describe('helpers/get-populated-export-contract/get-populated-agent', () => {
+  let context: Context;
+
+  let createdAgent: ApplicationExportContractAgent;
+  let createdService: ApplicationExportContractAgentService;
+  let createdServiceCharge: ApplicationExportContractAgentServiceCharge;
+
+  let serviceConnectObject = {};
+
+  beforeAll(async () => {
+    context = getKeystoneContext();
+  });
+
+  beforeEach(async () => {
+    createdService = (await service.create(context)) as ApplicationExportContractAgentService;
+
+    serviceConnectObject = {
+      connect: {
+        id: createdService.id,
+      },
+    };
+
+    const initServiceCharge = {
+      service: serviceConnectObject,
+    };
+
+    createdServiceCharge = (await serviceCharge.create(context, initServiceCharge)) as ApplicationExportContractAgentServiceCharge;
+
+    const initAgent = {
+      service: serviceConnectObject,
+    };
+
+    createdAgent = (await agent.create(context, initAgent)) as ApplicationExportContractAgent;
+  });
+
+  it('should return a populated agent', async () => {
+    const result = await getPopulatedAgent(context, createdAgent.id);
+
+    expect(result.id).toEqual(createdAgent.id);
+    expect(result.service.id).toEqual(createdService.id);
+    expect(result.service.charge.id).toEqual(createdServiceCharge.id);
+  });
+
+  describe('when an agent is not found', () => {
+    it('should throw an error', async () => {
+      const invalidId = 'invalid-id';
```

Comment:
const at the top? and in other files too as repeated

Reply:
Updated all instances of "invalid ID" stubs in the API